pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/SegmentConversionUtils.java

```diff
@@ -200,18 +201,21 @@ public static String startSegmentReplace(String tableNameWithType, String upload
     }
   }
 
-  public static void endSegmentReplace(String tableNameWithType, String uploadURL, String segmentLineageEntryId,
-      int socketTimeoutMs, @Nullable AuthProvider authProvider)
+  public static void endSegmentReplace(String tableNameWithType, String uploadURL,
```

Comment:
Let's add one extra function without `EndReplaceSegmentsRequest` to keep the backward compatibility

Reply:
Ok, added.